{
    "Information": {
        "Category": null, 
        "CAPEC": null, 
        "CVE": [
            {
                "url": "http://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2008-5031", 
                "published": "2008-11-10T11:15:12.140-05:00", 
                "id": "CVE-2008-5031", 
                "modified": "2012-10-30T23:06:35.357-04:00", 
                "summary": "Multiple integer overflows in Python 2.2.3 through 2.5.1, and 2.6, allow context-dependent attackers to have an unknown impact via a large integer value in the tabsize argument to the expandtabs method, as implemented by (1) the string_expandtabs function in Objects/stringobject.c and (2) the unicode_expandtabs function in Objects/unicodeobject.c.  NOTE: this vulnerability reportedly exists because of an incomplete fix for CVE-2008-2315."
            }
        ], 
        "CWE": [
            {
                "url": "https://cwe.mitre.org/data/definitions/189.html", 
                "id": "CWE-189", 
                "title": "Numeric Errors"
            }
        ], 
        "CPE": [
            {
                "id": "cpe:/a:python:python:2.6"
            }, 
            {
                "id": "cpe:/a:python:python:2.2.3"
            }, 
            {
                "id": "cpe:/a:python:python:2.5.1"
            }, 
            {
                "id": "cpe:/a:python:python:2.3.7"
            }, 
            {
                "id": "cpe:/a:python:python:2.4.6"
            }
        ]
    }, 
    "References": {
        "SCIP": [
            {
                "url": "http://www.scip.ch/?vuldb.44964", 
                "id": 44964
            }
        ], 
        "IAVM": [], 
        "BID": [], 
        "OSVDB": [], 
        "Other": {
            "References": [
                {
                    "url": "http://xforce.iss.net/xforce/xfdb/46612", 
                    "vendor": "XF"
                }, 
                {
                    "url": "http://www.vupen.com/english/advisories/2009/3316", 
                    "vendor": "VUPEN"
                }, 
                {
                    "url": "http://www.vmware.com/security/advisories/VMSA-2009-0016.html", 
                    "vendor": "CONFIRM"
                }, 
                {
                    "url": "http://www.securityfocus.com/archive/1/archive/1/507985/100/0/threaded", 
                    "vendor": "BUGTRAQ"
                }, 
                {
                    "url": "http://www.openwall.com/lists/oss-security/2008/11/05/3", 
                    "vendor": "MLIST"
                }, 
                {
                    "url": "http://www.openwall.com/lists/oss-security/2008/11/05/2", 
                    "vendor": "MLIST"
                }, 
                {
                    "url": "http://svn.python.org/view?rev=61350&view=rev", 
                    "vendor": "CONFIRM"
                }, 
                {
                    "url": "http://svn.python.org/view/python/trunk/Objects/unicodeobject.c?rev=61350&view=diff&r1=61350&r2=61349&p1=python/trunk/Objects/unicodeobject.c&p2=/python/trunk/Objects/unicodeobject.c", 
                    "vendor": "CONFIRM"
                }, 
                {
                    "url": "http://svn.python.org/view/python/trunk/Objects/stringobject.c?rev=61350&view=diff&r1=61350&r2=61349&p1=python/trunk/Objects/stringobject.c&p2=/python/trunk/Objects/stringobject.c", 
                    "vendor": "CONFIRM"
                }, 
                {
                    "url": "http://support.apple.com/kb/HT3438", 
                    "vendor": "CONFIRM"
                }, 
                {
                    "url": "http://security.gentoo.org/glsa/glsa-200907-16.xml", 
                    "vendor": "GENTOO"
                }, 
                {
                    "url": "http://secunia.com/advisories/37471", 
                    "vendor": "SECUNIA"
                }, 
                {
                    "url": "http://secunia.com/advisories/35750", 
                    "vendor": "SECUNIA"
                }, 
                {
                    "url": "http://secunia.com/advisories/33937", 
                    "vendor": "SECUNIA"
                }, 
                {
                    "url": "http://scary.beasts.org/security/CESA-2008-008.html", 
                    "vendor": "MISC"
                }, 
                {
                    "url": "http://lists.apple.com/archives/security-announce/2009/Feb/msg00000.html", 
                    "vendor": "APPLE"
                }
            ]
        }, 
        "CertVN": []
    }, 
    "Risk": [
        {
            "CVSS v2": [
                {
                    "impact": "10.0", 
                    "access complexity": "low", 
                    "confidentiality impact": "complete", 
                    "authentication": "none", 
                    "exploit": "10.0", 
                    "base": "10.0", 
                    "integrity impact": "complete", 
                    "availability impact": "complete", 
                    "access vector": "network"
                }
            ], 
            "severity": "High", 
            "Top alert": false, 
            "Top vulnerable": true
        }
    ], 
    "Rules": {
        "Suricata": [], 
        "Snort": []
    }, 
    "Exploits": {
        "Metasploit": [], 
        "ExploitDB": [], 
        "Elliot D2": [], 
        "Saint": []
    }, 
    "vFeed": {
        "Product": "vFeed - The Correlated Vulnerability and Threat Database", 
        "Contact": "@toolswatch", 
        "Author": "NJ OUCHN", 
        "URL": "https://github.com/toolswatch/vFeed", 
        "Version": "0.6.5", 
        "id": "VFD-2008-5031"
    }, 
    "Scanners": {
        "Oval": [
            {
                "url": "http://oval.mitre.org/repository/data/getDef?id=oval:org.mitre.oval:def:29294", 
                "title": "RHSA-2009:1176 -- python security update (Moderate)", 
                "class": "patch", 
                "id": "oval:org.mitre.oval:def:29294"
            }, 
            {
                "url": "http://oval.mitre.org/repository/data/getDef?id=oval:org.mitre.oval:def:11280", 
                "title": "Multiple integer overflows in Python 2.2.3 through 2.5.1, and 2.6, allow context-dependent attackers to have an unknown impact via a large integer value in the tabsize argument to the expandtabs method, as implemented by (1) the string_expandtabs function in Objects/stringobject.c and (2) the unicode_expandtabs function in Objects/unicodeobject.c.  NOTE: this vulnerability reportedly exists because of an incomplete fix for CVE-2008-2315.", 
                "class": "vulnerability", 
                "id": "oval:org.mitre.oval:def:11280"
            }, 
            {
                "url": "http://oval.mitre.org/repository/data/getDef?id=oval:org.mitre.oval:def:22809", 
                "title": "ELSA-2009:1176: python security update (Moderate)", 
                "class": "patch", 
                "id": "oval:org.mitre.oval:def:22809"
            }, 
            {
                "url": "http://oval.mitre.org/repository/data/getDef?id=oval:org.mitre.oval:def:8564", 
                "title": "VMware python multiple integer overflows vulnerability", 
                "class": "vulnerability", 
                "id": "oval:org.mitre.oval:def:8564"
            }, 
            {
                "url": "http://oval.mitre.org/repository/data/getDef?id=oval:org.mitre.oval:def:13081", 
                "title": "USN-806-1 -- python2.4, python2.5 vulnerabilities", 
                "class": "patch", 
                "id": "oval:org.mitre.oval:def:13081"
            }
        ], 
        "Nmap": [], 
        "OpenVas": [
            {
                "name": "CentOS Update for python CESA-2009:1176 centos5 i386", 
                "file": "gb_CESA-2009_1176_python_centos5_i386.nasl", 
                "family": "CentOS Local Security Checks", 
                "id": "880881"
            }, 
            {
                "name": "CentOS Update for python CESA-2009:1178 centos3 i386", 
                "file": "gb_CESA-2009_1178_python_centos3_i386.nasl", 
                "family": "CentOS Local Security Checks", 
                "id": "880715"
            }, 
            {
                "name": "Python Multiple Integer Overflow Vulnerabilities (Windows)", 
                "file": "gb_python_intgr_overflow_vuln_win.nasl", 
                "family": "Buffer overflow", 
                "id": "800056"
            }, 
            {
                "name": "Gentoo Security Advisory GLSA 200907-16 (python)", 
                "file": "glsa_200907_16.nasl", 
                "family": "Gentoo Local Security Checks", 
                "id": "64438"
            }, 
            {
                "name": "get_kb_item(ssh/login/osx_name", 
                "file": "macosx_secupd_2009-001.nasl", 
                "family": "Mac OS X Local Security Checks", 
                "id": "102026"
            }, 
            {
                "name": "Mandrake Security Advisory MDVSA-2009:003 (python)", 
                "file": "mdksa_2009_003.nasl", 
                "family": "Mandrake Local Security Checks", 
                "id": "63136"
            }, 
            {
                "name": "Mandrake Security Advisory MDVSA-2009:036 (python)", 
                "file": "mdksa_2009_036.nasl", 
                "family": "Mandrake Local Security Checks", 
                "id": "63373"
            }, 
            {
                "name": "CentOS Security Advisory CESA-2009:1176 (python)", 
                "file": "ovcesa2009_1176.nasl", 
                "family": "CentOS Local Security Checks", 
                "id": "64583"
            }, 
            {
                "name": "CentOS Security Advisory CESA-2009:1178 (python)", 
                "file": "ovcesa2009_1178.nasl", 
                "family": "CentOS Local Security Checks", 
                "id": "64493"
            }, 
            {
                "name": "RedHat Security Advisory RHSA-2009:1176", 
                "file": "RHSA_2009_1176.nasl", 
                "family": "Red Hat Local Security Checks", 
                "id": "64453"
            }, 
            {
                "name": "RedHat Security Advisory RHSA-2009:1177", 
                "file": "RHSA_2009_1177.nasl", 
                "family": "Red Hat Local Security Checks", 
                "id": "64454"
            }, 
            {
                "name": "RedHat Security Advisory RHSA-2009:1178", 
                "file": "RHSA_2009_1178.nasl", 
                "family": "Red Hat Local Security Checks", 
                "id": "64455"
            }, 
            {
                "name": "SLES10: Security update for Python", 
                "file": "sles10_python0.nasl", 
                "family": "SuSE Local Security Checks", 
                "id": "65933"
            }, 
            {
                "name": "SLES9: Security update for Python", 
                "file": "sles9p5040780.nasl", 
                "family": "SuSE Local Security Checks", 
                "id": "65048"
            }, 
            {
                "name": "SuSE Security Summary SUSE-SR:2009:001 (OpenSuSE 11.1)", 
                "file": "suse_sr_2009_001.nasl", 
                "family": "SuSE Local Security Checks", 
                "id": "63186"
            }, 
            {
                "name": "SuSE Security Summary SUSE-SR:2009:001 (OpenSuSE 11.0)", 
                "file": "suse_sr_2009_001a.nasl", 
                "family": "SuSE Local Security Checks", 
                "id": "63187"
            }, 
            {
                "name": "SuSE Security Summary SUSE-SR:2009:001 (OpenSuSE 10.3)", 
                "file": "suse_sr_2009_001b.nasl", 
                "family": "SuSE Local Security Checks", 
                "id": "63188"
            }, 
            {
                "name": "Ubuntu USN-806-1 (python2.5)", 
                "file": "ubuntu_806_1.nasl", 
                "family": "Ubuntu Local Security Checks", 
                "id": "64488"
            }
        ], 
        "Nessus": [
            {
                "name": "CentOS 5 : python (CESA-2009:1176)", 
                "file": "centos_RHSA-2009-1176.nasl", 
                "family": "CentOS Local Security Checks", 
                "id": "43771"
            }, 
            {
                "name": "CentOS 3 : python (CESA-2009:1178)", 
                "file": "centos_RHSA-2009-1178.nasl", 
                "family": "CentOS Local Security Checks", 
                "id": "40394"
            }, 
            {
                "name": "GLSA-200907-16 : Python: Integer overflows", 
                "file": "gentoo_GLSA-200907-16.nasl", 
                "family": "Gentoo Local Security Checks", 
                "id": "39870"
            }, 
            {
                "name": "Mac OS X Multiple Vulnerabilities (Security Update 2009-001)", 
                "file": "macosx_SecUpd2009-001.nasl", 
                "family": "MacOS X Local Security Checks", 
                "id": "35684"
            }, 
            {
                "name": "Mandriva Linux Security Advisory : python (MDVSA-2009:003)", 
                "file": "mandriva_MDVSA-2009-003.nasl", 
                "family": "Mandriva Local Security Checks", 
                "id": "36693"
            }, 
            {
                "name": "Oracle Linux 5 : python (ELSA-2009-1176)", 
                "file": "oraclelinux_ELSA-2009-1176.nasl", 
                "family": "Oracle Linux Local Security Checks", 
                "id": "67896"
            }, 
            {
                "name": "Oracle Linux 4 : python (ELSA-2009-1177)", 
                "file": "oraclelinux_ELSA-2009-1177.nasl", 
                "family": "Oracle Linux Local Security Checks", 
                "id": "67897"
            }, 
            {
                "name": "Oracle Linux 3 : python (ELSA-2009-1178)", 
                "file": "oraclelinux_ELSA-2009-1178.nasl", 
                "family": "Oracle Linux Local Security Checks", 
                "id": "67898"
            }, 
            {
                "name": "RHEL 5 : python (RHSA-2009:1176)", 
                "file": "redhat-RHSA-2009-1176.nasl", 
                "family": "Red Hat Local Security Checks", 
                "id": "40400"
            }, 
            {
                "name": "RHEL 4 : python (RHSA-2009:1177)", 
                "file": "redhat-RHSA-2009-1177.nasl", 
                "family": "Red Hat Local Security Checks", 
                "id": "40401"
            }, 
            {
                "name": "RHEL 3 : python (RHSA-2009:1178)", 
                "file": "redhat-RHSA-2009-1178.nasl", 
                "family": "Red Hat Local Security Checks", 
                "id": "40402"
            }, 
            {
                "name": "Scientific Linux Security Update : python for SL5.x i386/x86_64", 
                "file": "sl_20090727_python_for_SL5_x.nasl", 
                "family": "Scientific Linux Local Security Checks", 
                "id": "60622"
            }, 
            {
                "name": "Scientific Linux Security Update : python for SL 3.0.x on i386/x86_64", 
                "file": "sl_20090728_python_for_SL_3_0_x.nasl", 
                "family": "Scientific Linux Local Security Checks", 
                "id": "60624"
            }, 
            {
                "name": "Scientific Linux Security Update : python for SL 4.x on i386/x86_64", 
                "file": "sl_20090728_python_for_SL_4_x.nasl", 
                "family": "Scientific Linux Local Security Checks", 
                "id": "60625"
            }, 
            {
                "name": "SuSE9 Security Update : Python (YOU Patch Number 12316)", 
                "file": "suse9_12316.nasl", 
                "family": "SuSE Local Security Checks", 
                "id": "41260"
            }, 
            {
                "name": "SuSE 10 Security Update : Python (ZYPP Patch Number 5837)", 
                "file": "suse_python-5837.nasl", 
                "family": "SuSE Local Security Checks", 
                "id": "41581"
            }, 
            {
                "name": "Ubuntu 6.06 LTS / 7.04 / 7.10 / 8.04 LTS : python2.4, python2.5 vulnerabilities (USN-632-1)", 
                "file": "ubuntu_USN-632-1.nasl", 
                "family": "Ubuntu Local Security Checks", 
                "id": "33807"
            }, 
            {
                "name": "Ubuntu 6.06 LTS / 8.04 LTS / 8.10 : python2.4, python2.5 vulnerabilities (USN-806-1)", 
                "file": "ubuntu_USN-806-1.nasl", 
                "family": "Ubuntu Local Security Checks", 
                "id": "40361"
            }, 
            {
                "name": "VMSA-2009-0016 : VMware vCenter and ESX update release and vMA patch release address multiple security issues in third party components.", 
                "file": "vmware_VMSA-2009-0016.nasl", 
                "family": "VMware ESX Local Security Checks", 
                "id": "42870"
            }, 
            {
                "name": "VMware ESX / ESXi Multiple Vulnerabilities (VMSA-2009-0016) (remote check)", 
                "file": "vmware_VMSA-2009-0016_remote.nasl", 
                "family": "Misc.", 
                "id": "89117"
            }
        ]
    }, 
    "Patches": {
        "Fedora": [], 
        "HP": [], 
        "Microsoft KB": [], 
        "Mandriva": [], 
        "Cisco": [], 
        "IBM AIX Apar": [], 
        "Suse": [], 
        "Redhat": [
            {
                "Redhat": {
                    "oval": "oval:com.redhat.rhsa:def:20091176", 
                    "url": "https://rhn.redhat.com/errata/RHSA-2009-1176.html", 
                    "id": "RHSA-2009:1176", 
                    "title": "RHSA-2009:1176: python security update (Moderate)"
                }
            }, 
            {
                "bugzilla": {
                    "url": "https://bugzilla.redhat.com/show_bug.cgi?id=235093", 
                    "title": "CVE-2007-2052 python off-by-one locale.strxfrm() (possible memory disclosure)", 
                    "id": "235093", 
                    "issue date": "2009-07-27", 
                    "associated redhat": "RHSA-2009:1176"
                }
            }, 
            {
                "bugzilla": {
                    "url": "https://bugzilla.redhat.com/show_bug.cgi?id=295971", 
                    "title": "CVE-2007-4965 python imageop module heap corruption", 
                    "id": "295971", 
                    "issue date": "2009-07-27", 
                    "associated redhat": "RHSA-2009:1176"
                }
            }, 
            {
                "bugzilla": {
                    "url": "https://bugzilla.redhat.com/show_bug.cgi?id=442005", 
                    "title": "CVE-2008-1721 python: integer signedness error in the zlib extension module", 
                    "id": "442005", 
                    "issue date": "2009-07-27", 
                    "associated redhat": "RHSA-2009:1176"
                }
            }, 
            {
                "bugzilla": {
                    "url": "https://bugzilla.redhat.com/show_bug.cgi?id=443810", 
                    "title": "CVE-2008-1887 python: PyString_FromStringAndSize does not check for negative size values", 
                    "id": "443810", 
                    "issue date": "2009-07-27", 
                    "associated redhat": "RHSA-2009:1176"
                }
            }, 
            {
                "bugzilla": {
                    "url": "https://bugzilla.redhat.com/show_bug.cgi?id=454990", 
                    "title": "CVE-2008-3142 python: Multiple buffer overflows in unicode processing", 
                    "id": "454990", 
                    "issue date": "2009-07-27", 
                    "associated redhat": "RHSA-2009:1176"
                }
            }, 
            {
                "bugzilla": {
                    "url": "https://bugzilla.redhat.com/show_bug.cgi?id=455008", 
                    "title": "CVE-2008-2315 python: Multiple integer overflows in python core", 
                    "id": "455008", 
                    "issue date": "2009-07-27", 
                    "associated redhat": "RHSA-2009:1176"
                }
            }, 
            {
                "bugzilla": {
                    "url": "https://bugzilla.redhat.com/show_bug.cgi?id=455013", 
                    "title": "CVE-2008-3143 python: Multiple integer overflows discovered by Google", 
                    "id": "455013", 
                    "issue date": "2009-07-27", 
                    "associated redhat": "RHSA-2009:1176"
                }
            }, 
            {
                "bugzilla": {
                    "url": "https://bugzilla.redhat.com/show_bug.cgi?id=455018", 
                    "title": "CVE-2008-3144 python: Potential integer underflow and overflow in the PyOS_vsnprintf C API function", 
                    "id": "455018", 
                    "issue date": "2009-07-27", 
                    "associated redhat": "RHSA-2009:1176"
                }
            }, 
            {
                "bugzilla": {
                    "url": "https://bugzilla.redhat.com/show_bug.cgi?id=469656", 
                    "title": "CVE-2008-4864 python: imageop module multiple integer overflows", 
                    "id": "469656", 
                    "issue date": "2009-07-27", 
                    "associated redhat": "RHSA-2009:1176"
                }
            }, 
            {
                "bugzilla": {
                    "url": "https://bugzilla.redhat.com/show_bug.cgi?id=470915", 
                    "title": "CVE-2008-5031 python: stringobject, unicodeobject integer overflows", 
                    "id": "470915", 
                    "issue date": "2009-07-27", 
                    "associated redhat": "RHSA-2009:1176"
                }
            }, 
            {
                "Redhat": {
                    "oval": "oval:com.redhat.rhsa:def:20091177", 
                    "url": "https://rhn.redhat.com/errata/RHSA-2009-1177.html", 
                    "id": "RHSA-2009:1177", 
                    "title": "RHSA-2009:1177: python security update (Moderate)"
                }
            }, 
            {
                "bugzilla": {
                    "url": "https://bugzilla.redhat.com/show_bug.cgi?id=441306", 
                    "title": "CVE-2008-1679 python: imageop module integer overflows", 
                    "id": "441306", 
                    "issue date": "2009-07-27", 
                    "associated redhat": "RHSA-2009:1177"
                }
            }, 
            {
                "bugzilla": {
                    "url": "https://bugzilla.redhat.com/show_bug.cgi?id=442005", 
                    "title": "CVE-2008-1721 python: integer signedness error in the zlib extension module", 
                    "id": "442005", 
                    "issue date": "2009-07-27", 
                    "associated redhat": "RHSA-2009:1177"
                }
            }, 
            {
                "bugzilla": {
                    "url": "https://bugzilla.redhat.com/show_bug.cgi?id=443810", 
                    "title": "CVE-2008-1887 python: PyString_FromStringAndSize does not check for negative size values", 
                    "id": "443810", 
                    "issue date": "2009-07-27", 
                    "associated redhat": "RHSA-2009:1177"
                }
            }, 
            {
                "bugzilla": {
                    "url": "https://bugzilla.redhat.com/show_bug.cgi?id=454990", 
                    "title": "CVE-2008-3142 python: Multiple buffer overflows in unicode processing", 
                    "id": "454990", 
                    "issue date": "2009-07-27", 
                    "associated redhat": "RHSA-2009:1177"
                }
            }, 
            {
                "bugzilla": {
                    "url": "https://bugzilla.redhat.com/show_bug.cgi?id=455008", 
                    "title": "CVE-2008-2315 python: Multiple integer overflows in python core", 
                    "id": "455008", 
                    "issue date": "2009-07-27", 
                    "associated redhat": "RHSA-2009:1177"
                }
            }, 
            {
                "bugzilla": {
                    "url": "https://bugzilla.redhat.com/show_bug.cgi?id=455013", 
                    "title": "CVE-2008-3143 python: Multiple integer overflows discovered by Google", 
                    "id": "455013", 
                    "issue date": "2009-07-27", 
                    "associated redhat": "RHSA-2009:1177"
                }
            }, 
            {
                "bugzilla": {
                    "url": "https://bugzilla.redhat.com/show_bug.cgi?id=455018", 
                    "title": "CVE-2008-3144 python: Potential integer underflow and overflow in the PyOS_vsnprintf C API function", 
                    "id": "455018", 
                    "issue date": "2009-07-27", 
                    "associated redhat": "RHSA-2009:1177"
                }
            }, 
            {
                "bugzilla": {
                    "url": "https://bugzilla.redhat.com/show_bug.cgi?id=469656", 
                    "title": "CVE-2008-4864 python: imageop module multiple integer overflows", 
                    "id": "469656", 
                    "issue date": "2009-07-27", 
                    "associated redhat": "RHSA-2009:1177"
                }
            }, 
            {
                "bugzilla": {
                    "url": "https://bugzilla.redhat.com/show_bug.cgi?id=470915", 
                    "title": "CVE-2008-5031 python: stringobject, unicodeobject integer overflows", 
                    "id": "470915", 
                    "issue date": "2009-07-27", 
                    "associated redhat": "RHSA-2009:1177"
                }
            }, 
            {
                "Redhat": {
                    "oval": "oval:com.redhat.rhsa:def:20091178", 
                    "url": "https://rhn.redhat.com/errata/RHSA-2009-1178.html", 
                    "id": "RHSA-2009:1178", 
                    "title": "RHSA-2009:1178: python security update (Moderate)"
                }
            }, 
            {
                "bugzilla": {
                    "url": "https://bugzilla.redhat.com/show_bug.cgi?id=441306", 
                    "title": "CVE-2008-1679 python: imageop module integer overflows", 
                    "id": "441306", 
                    "issue date": "2009-07-27", 
                    "associated redhat": "RHSA-2009:1178"
                }
            }, 
            {
                "bugzilla": {
                    "url": "https://bugzilla.redhat.com/show_bug.cgi?id=443810", 
                    "title": "CVE-2008-1887 python: PyString_FromStringAndSize does not check for negative size values", 
                    "id": "443810", 
                    "issue date": "2009-07-27", 
                    "associated redhat": "RHSA-2009:1178"
                }
            }, 
            {
                "bugzilla": {
                    "url": "https://bugzilla.redhat.com/show_bug.cgi?id=454990", 
                    "title": "CVE-2008-3142 python: Multiple buffer overflows in unicode processing", 
                    "id": "454990", 
                    "issue date": "2009-07-27", 
                    "associated redhat": "RHSA-2009:1178"
                }
            }, 
            {
                "bugzilla": {
                    "url": "https://bugzilla.redhat.com/show_bug.cgi?id=455008", 
                    "title": "CVE-2008-2315 python: Multiple integer overflows in python core", 
                    "id": "455008", 
                    "issue date": "2009-07-27", 
                    "associated redhat": "RHSA-2009:1178"
                }
            }, 
            {
                "bugzilla": {
                    "url": "https://bugzilla.redhat.com/show_bug.cgi?id=455013", 
                    "title": "CVE-2008-3143 python: Multiple integer overflows discovered by Google", 
                    "id": "455013", 
                    "issue date": "2009-07-27", 
                    "associated redhat": "RHSA-2009:1178"
                }
            }, 
            {
                "bugzilla": {
                    "url": "https://bugzilla.redhat.com/show_bug.cgi?id=455018", 
                    "title": "CVE-2008-3144 python: Potential integer underflow and overflow in the PyOS_vsnprintf C API function", 
                    "id": "455018", 
                    "issue date": "2009-07-27", 
                    "associated redhat": "RHSA-2009:1178"
                }
            }, 
            {
                "bugzilla": {
                    "url": "https://bugzilla.redhat.com/show_bug.cgi?id=469656", 
                    "title": "CVE-2008-4864 python: imageop module multiple integer overflows", 
                    "id": "469656", 
                    "issue date": "2009-07-27", 
                    "associated redhat": "RHSA-2009:1178"
                }
            }, 
            {
                "bugzilla": {
                    "url": "https://bugzilla.redhat.com/show_bug.cgi?id=470915", 
                    "title": "CVE-2008-5031 python: stringobject, unicodeobject integer overflows", 
                    "id": "470915", 
                    "issue date": "2009-07-27", 
                    "associated redhat": "RHSA-2009:1178"
                }
            }
        ], 
        "Gentoo": [
            {
                "url": "https://security.gentoo.org/glsa/200907-16", 
                "id": "GLSA-200907-16"
            }
        ], 
        "Microsoft Bulletins": [], 
        "Ubuntu": [], 
        "Debian": [], 
        "Vmware": [
            {
                "url": "https://www.vmware.com/security/advisories/VMSA-2009-0016.html", 
                "id": "VMSA-2009-0016"
            }
        ]
    }
}